notebooks/bimodal_dist_plot.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    n = [norm(loc=mu[i], scale=sigma[i]) for i in range(2)]
</pre><p>What is n? if these are samples, rename n with samples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thease are distributions so better rename as distributions.